### PR TITLE
Fix bug and add generation to action

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
@MaxGhenis I found the bug affecting the Child Benefit parameters: the `role` field was inverted, I think since the pandas rewrite. The reason this didn't affect anything else is because in openfisca-uk, I've been moving away from using the role syntax as much as possible, preferring to use the variable `is_child` which explicitly calculates based on age. I've also added a generation action here, which updates the FRS dataset in Google Cloud Storage (also in future to generate FRS 2018, as well as the WAS imputation).

### CODE REVIEW

File: openfisca_uk_data/datasets/frs/frs.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def add_personal_variables(frs: h5py.File, person: DataFrame):
     # Add basic personal variables
     age = person.AGE80 + person.AGE
     frs["age"] = age
-    frs["role"] = np.where(person.AGE80 == 0, "adult", "child").astype("S")
+    frs["role"] = np.where(person.AGE80 == 0, "child", "adult").astype("S")
```
Comment (MaxGhenis) [inline openfisca_uk_data/datasets/frs/frs.py:157]: Could you add a comment on what `role` and `AGE80` are?

**Review by MaxGhenis — APPROVED** [review]

File: openfisca_uk_data/datasets/frs/frs.py (hunk 1 above)
Comment (nikhilwoodruff) [reply openfisca_uk_data/datasets/frs/frs.py:160]: Added

Comment (MaxGhenis) [reply openfisca_uk_data/datasets/frs/frs.py:160]: Thanks, just to be explicit could you add one more sentence (if I'm understanding it correctly):
```
# AGE80 == 0 therefore indicates that the record is from the child table.
```

Comment (nikhilwoodruff) [reply openfisca_uk_data/datasets/frs/frs.py:160]: Ah no problem, will add that

**Pull request merged**